tests/v2/validation/snapshot/snapshot.go

```diff
@@ -229,56 +220,45 @@ func restoreRKE1(t *testing.T, client *rancher.Client, rke1Snapshot initialSnaps
 
 		nodestat.AllManagementNodeReady(client, clusterResp.ID, extdefault.ThirtyMinuteTimeout)
 
+		client, err = client.ReLogin()
+		require.NoError(t, err)
+
 		podErrors := pods.StatusPods(client, clusterID)
 		assert.Empty(t, podErrors)
-		require.Equal(t, rke1Snapshot.kubernetesVersion, clusterResp.RancherKubernetesEngineConfig.Version)
 
-		if etcdRestore.SnapshotRestore == kubernetesVersion || etcdRestore.SnapshotRestore == all {
-			clusterResp, err = client.Management.Cluster.ByID(clusterID)
-			require.NoError(t, err)
-			require.Equal(t, rke1Snapshot.kubernetesVersion, clusterResp.RancherKubernetesEngineConfig.Version)
+		require.Equal(t, cluster.RancherKubernetesEngineConfig.Version, clusterResp.RancherKubernetesEngineConfig.Version)
 
-			if etcdRestore.ControlPlaneUnavailableValue != "" && etcdRestore.WorkerUnavailableValue != "" {
-				logrus.Infof("Control plane unavailable value is restored to: %s", clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane)
-				logrus.Infof("Worker unavailable value is restored to: %s", clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker)
+		if etcdRestore.ControlPlaneUnavailableValue != "" && etcdRestore.WorkerUnavailableValue != "" {
+			logrus.Infof("Control plane unavailable value is set to: %s", cluster.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane)
```

Comment:
I think `cluster` should be named something like `oldClusterObject` or something of the like. Reading through this is a little confusing, because we normally use `cluster` to represent the current version of the object. 

Which I think is what you meant/need to do here, technically this should be set to clusterResp at this stage because we haven't checked that they are the same value(s) yet.

Reply:
Okay, I'll change it to `oldCluster`.

Reply:
Updated the name.